ReactiveExample/LoginViewModel.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Reactive;
+using System.Reactive.Linq;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+using NStack;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+namespace ReactiveExample {
+	//
+	// This view model can be easily shared across different UI frameworks.
```

Comment:
This gives me warm feels.

Not for this PR, but a pointer to a sample or real-world app that does this would be awesome (I mean, *I* really would love to have one because I'm currently building a XAML app that I'd love to port to Terminal.Gui).

Reply:
We have a few samples listed on this page https://www.reactiveui.net/docs/resources/samples Also, here is a tiny and cross-platform ReactiveUI sample app with a lot of frontends, including Terminal.Gui as well https://github.com/worldbeater/reactivemvvm